server.js

```diff
@@ -1,32 +1,30 @@
-import express from 'express'
+import express, { response } from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import data from './data/netflix-titles.json'
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
 //   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/year', (req, res) => {
+  res.json(data)
+})
+app.get('/years/:year', (req, res) => {
+  const year = req.params.year
+  const releaseYear = data.filter((item) => item.release_year === +year)
+
+    res.json(releaseYear)
+})
```

Comment:
Nice with some filtering 👍 
A suggestion would be to make this a query param under your first endpoint, like:
`/shows?year=2020`